llvm/test/CodeGen/DirectX/llc-pipeline.ll

```diff
@@ -31,6 +31,7 @@
 ; CHECK-NEXT:   DXIL Module Metadata analysis
 ; CHECK-NEXT:   DXIL Shader Flag Analysis
 ; CHECK-NEXT:   DXIL Translate Metadata
+; CHECK-NEXT:   DXIL Root Signature Analysis
```

Comment:
This shouldn't be running twice. As per https://github.com/llvm/llvm-project/pull/146783#pullrequestreview-2998542245 we need to make sure the analysis is preserved instead.